INN/umbrella-workdayminn
# Homepage top story layout

## Changes

This pull request makes the following changes:

- Added styling/layout for the homepage top story layout
- Created mailchimp signup form partial to reuse

**With image:**

Desktop:
![Screen Shot 2019-08-07 at 10 06 41 AM](https://user-images.githubusercontent.com/18353636/62629494-3959cf80-b8fb-11e9-86e7-d79ca63e6b9b.png)

Mobile:
![Screen Shot 2019-08-07 at 10 07 00 AM](https://user-images.githubusercontent.com/18353636/62629493-38c13900-b8fb-11e9-912b-ab46716b5f09.png)

**Without image:**

Desktop:
![Screen Shot 2019-08-07 at 10 15 12 AM](https://user-images.githubusercontent.com/18353636/62630098-52af4b80-b8fc-11e9-94d1-15cd031e8edc.png)

Mobile:
![Screen Shot 2019-08-07 at 10 15 00 AM](https://user-images.githubusercontent.com/18353636/62630099-52af4b80-b8fc-11e9-870c-150e5e48e9e9.png)

## Why



For #12

## Testing/Questions

Features that this PR affects:

- Homepage top story + newsletter sign up


Questions that need to be answered before merging:

- [ ] Are all mailchimp edge cases covered styling wise?

Steps to test this PR:

1. Set a post as the homepage top story
2. Verify it looks ok with a featured image
3. Verify it looks ok without a featured image
4. Sign up to the newsletter and verify that works and looks ok
